packages/components/src/mobile/segmented-control/index.native.js

```diff
@@ -113,16 +113,10 @@ const SegmentedControls = ( {
 	}
 
 	function onHandlePress( segment, index ) {
-		LayoutAnimation.configureNext(
-			LayoutAnimation.create(
-				ANIMATION_DURATION,
-				LayoutAnimation.Types.easeInEaseOut,
-				LayoutAnimation.Properties.opacity
-			)
-		);
+		performLayoutAnimation( ANIMATION_DURATION );
 		setActiveSegmentIndex( index );
 		segmentHandler( segment );
-		performAnimation( index, segment );
+		performSwatchAnimation( index, segment );
```

Comment:
`performSwatchAnimation` doesn't look like it accepts a second argument.

Reply:
Thanks, it's leftover after clean up.